ep008/README.md

```diff
@@ -0,0 +1,34 @@
+# [what's wrong with python's blank except:? (beginner - intermediate)](https://youtu.be/i0GOLe-F27Q)
+
+The dreaded "E722 do not use bare 'except'"! Today we explain the various ways to "catch all exceptions" as well as some recommendations on how to properly fix this problem!
+
+## Setup commands
+
+```bash
+virtualenv venv
+
+. venv/bin/activate
+
+pip install flake8
+```
+
+## Interactive examples
+
+### Python
+
+```python
+# python 3
+Exception.__bases__
+BaseException.__bases__
+BaseException.__subclasses__()
+```
+
+### Bash
+
+```bash
+echo hello world > dne
+# echo 'hello world' > dne
```

Comment:
Added this as alternative example command, since the Windows version of `echo` will add new lines between the words if you don't use quotes :D